### ISSUE TITLE
Jump to Source

### ISSUE THREAD

**Issue by NightRa** [carymrobbins/intellij-haskforce#192]
A feature I really miss in Haskell IDEs is jumping to the source of functions outside the project.
This can be acheived via the ```codex``` package.

**Comment by carymrobbins** [comment]
Agreed.  There is a way to get it to work now, though.  You can do this by using `cabal get`.  I usually create a `lib` dir and just put the sources there.  However, true codex (or similar) integration and support would be really nice.

**Comment by carymrobbins** [comment]
Now that I'm implementing stack support, I'm wondering how this would work with stack.  Possibly we could still use `cabal get`, but I'd be worried about how it may affect the stack project, as well as requiring the user to have a `cabal` executable configured.

If anyone knows of a way to use stack (or similar tooling) to achieve this _without_ cabal, that'd be great.  All we need to do is be able to download the library sources.  The IntelliJ indexer will take care of the rest.

**Comment by carymrobbins** [comment]
`stack unpack` will actually work for this.

**Comment by KadekM** [comment]
Here are step by step instructions: https://github.com/carymrobbins/intellij-haskforce/wiki/Quick-Start-Guide#jump-to-library-sources

**Comment by johnwu93** [comment]
I actually tried doing this. I created a `lib` folder and unpacked `scotty` to it. The `GoTo` method still does not work for `scotty` methods
